apps/mobile-wallet/ios/Podfile.lock

```diff
@@ -1618,7 +1618,7 @@ PODS:
     - React-Core
   - RNFlashList (1.6.4):
     - React-Core
-  - RNGestureHandler (2.16.2):
+  - RNGestureHandler (2.21.2):
     - DoubleConversion
     - glog
```

Comment:
small gitch on Android with the bottom padding:

<img width="350" alt="image" src="https://github.com/user-attachments/assets/716cdff8-08bf-465a-a36d-a4d510a62afe">

Reply:
This issue is fixed in https://github.com/alephium/alephium-frontend/pull/1032

Closing